package.json

```diff
@@ -21,5 +22,9 @@
       "mustache": "github:janl/mustache.js/0.7.2",
       "text": "github:requirejs/text"
     }
+  },
+  "dependencies": {
+    "mustache": "^4.2.0",
```

Comment:
Why is mustache changed in dependencies and yet not five lines above?

Reply:
We will need it in dev environment , testing grunt with the current library wasn't possible and as I had to add extra libraries the package.json added it itself. We need it to test with karma and it wasn't getting installed when just doing npm install for the reasons I am not aware of, I think it would be better if we keep it in devDependencies . Especially about webL10n it's no longer getting installed via ```npm install webL10n``` so we need the following one. And in the following way loader.js is made such that it recognizes the libraries after installation.